src/lib/storage/segment_iterables/any_segment_iterator.hpp

```diff
@@ -113,7 +113,7 @@ class AnySegmentIterator : public BaseSegmentIterator<AnySegmentIterator<T>, Seg
   SegmentIteratorValue<T> dereference() const { return _wrapper->dereference(); }
 
  private:
-  std::unique_ptr<detail::AnySegmentIteratorWrapperBase<T>> _wrapper;
+  std::unique_ptr<opossum::detail::AnySegmentIteratorWrapperBase<T>> _wrapper;
```

Comment:
Accident or really necessary?

Reply:
Necessary, even though I can't really tell you why.